schema/objects/font-data-ref.schema.yaml

```diff
@@ -0,0 +1,17 @@
+title: Font Data Ref
+description: Defines a reference to base64 font data within the document bundle
+type: object
+properties:
+  _class: { const: MSJSONOriginalDataReference }
+  _ref_class: { const: MSEmbeddedFontData }
```

Comment:
Should this be `embeddedFontData` and a replacement in `SketchModel` similar to `embeddedFontReference` or does this not apply to `_ref_class`?

Reply:
@christianklotz EmbeddedFontReference has an EmbeddedFontData similar to ImageData.

Reply:
I think @jnozzi wanted to keep the implementation of this part as close as possible to image data references, which use `MSImageData`.

Reply:
And yeah, looking MSJSONArchiver it doesn't seem as those the replacements are setup to work with `_ref_class` values ...

Reply:
👍